.github/workflows/python-app.yml

```diff
@@ -13,6 +13,8 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    env:
```

Comment:
@ObaraEmmanuel Remove this. I have added the SECRET_KEY to the GITHUB SECRETS

Reply:
let me replace it with the github secret. The `env` has to be set either way